include/libsemigroups/sims.hpp

```diff
@@ -1801,7 +1778,16 @@ namespace libsemigroups {
     RepOrc() : _min(), _max(), _size() {
       init();
     }
-    // TODO(doc)
+
+    //! \brief Reinitialize an existing RepOrc object.
+    //!
+    //! This function puts a RepOrc object back into the same state as if
+    //! it had been newly default constructed.
+    //!
+    //! \returns A reference to \c this.
+    //!
+    //! \exception
```

Comment:
```suggestion
    //! \exceptions
```

See #628 for more info.